.ado/azure-pipelines.yml

```diff
@@ -1,90 +1,121 @@
 pr:
   - master
 
-pool:
-  vmImage: 'macos-10.15'
-
 variables:
   CI: true
 
 jobs:
+  - job: WindowsPR
+    displayName: Windows PR
+    pool:
+      vmImage: "windows-2019"
+    timeoutInMinutes: 60 # how long to run the job before automatically cancelling
+    cancelTimeoutInMinutes: 5 # how much time to give 'run always even if cancelled tasks' before killing them
+
+    steps:
+      - checkout: self
+        persistCredentials: true
+
+      - task: NodeTool@0
+        inputs:
+          versionSpec: "10.x"
```

Comment:
I'm not opposed to the double quotes per se, but is there a good reason for it? I think there are a few small differences between single quotes, double quotes, and no quotes that I doubt will affect us, but still worth considering.

Reply:
I followed the pattern from the Apple job below. I have no strong preference for either.

Reply:
I did some research on this - it doesn't look like the difference really effects us.